tools/BuildAssets/BuildTasks/Microsoft.Azure.Sdk.Build.Tasks/BootstrapTasks/Microsoft.Azure.Build.BootstrapTasks.csproj

```diff
@@ -20,18 +21,12 @@
   <PropertyGroup>
     <TargetFramework>net46</TargetFramework>
     <OutputPath>$(TaskBinaryOutput)</OutputPath>
+    <DelaySign>false</DelaySign>
```

Comment:
You shouldn't need this because you set it in buildtools.props

Reply:
@wedebols good catch, will remove this.